frontend/src/modules/conversation/components/conversation-platform-tabs.vue

```diff
@@ -93,6 +107,15 @@ export default {
     &--active {
```

Comment:
Same thing for this one, if we won't be using it anymore, let's just remove it 👍

Reply:
Same as above :)